### ISSUE TITLE
Not detecting lines correctly, even after 2100 epoch, dataset attached

### ISSUE THREAD

**Issue by ghost** [BADBADBADBOY/pytorch.ctpn#5]
@BADBADBADBOY 
When training on a text-line dataset, download here: [dataset download](https://drive.google.com/file/d/1YkXwgUNvcoYc_dSKnauKmRCKjBTsM0S6/view?usp=sharing)
[ctpn_2100.pth
](https://drive.google.com/file/d/10LnszFpDgLrR9mNJSYdFZQ0raPGtEks_/view?usp=sharing)
It can't detect the text-lines correctly, even after training for 2100 epochs.
Waiting for your reply

Sample results:
![001_result](https://user-images.githubusercontent.com/49877135/72110097-73b31700-3348-11ea-9e12-151bb5186c20.jpg)
![011_result](https://user-images.githubusercontent.com/49877135/72110115-7ada2500-3348-11ea-9b37-ab2ddff14e28.jpg)

**Comment by BADBADBADBOY** [comment]
@deepseek  The picture you gave can't be displayed, whether the training is normal or not, and whether loss drops normally or not.

**Comment by BADBADBADBOY** [comment]
@deepseek Have you tested the model I provided, and can it detect the text normally?

**Comment by ghost** [comment]
I chaged the `im_scale` to 1000, now getting somewhat better detection, but still ctpn have alot of downfalls
